sdk/spring/spring-cloud-azure-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -38,7 +38,8 @@ com.azure.spring.cloud.autoconfigure.resourcemanager.AzureEventHubsResourceManag
 com.azure.spring.cloud.autoconfigure.resourcemanager.AzureServiceBusResourceManagerAutoConfiguration,\
 com.azure.spring.cloud.autoconfigure.resourcemanager.AzureStorageQueueResourceManagerAutoConfiguration,\
 com.azure.spring.cloud.autoconfigure.compatibility.AzureCompatibilityVerifierAutoConfiguration,\
-com.azure.spring.cloud.autoconfigure.jdbc.AzureJdbcAutoConfiguration
+com.azure.spring.cloud.autoconfigure.jdbc.AzureJdbcAutoConfiguration,\
+com.azure.spring.cloud.autoconfigure.redis.passwordless.jedis.AzureJedisPasswordlessAutoConfiguration
```

Comment:
Will `com.azure.spring.cloud.autoconfigure.redis.passwordless.AzureJedisPasswordlessAutoConfiguration
` be better?

Reply:
Or I think we could remove the passwordless from the package name, since we already have that in the classname.

Reply:
Agree